flank-scripts/build.gradle.kts

```diff
@@ -199,5 +181,3 @@ fun execAndGetStdout(vararg args: String): String {
     }
     return stdout.toString().trimEnd()
 }
-
-tasks["detekt"].dependsOn(tasks["checkIfVersionUpdated"])
```

Comment:
However, this should not be removed, only `detekt` should be renamed to new linter task name

Reply:
@piotradamczyk5 are you sure. By adding this dependancy it will break the build.
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flank-scripts:checkIfVersionUpdated'.
> Flank scripts version is not updated, but files changed.
  Please update version according to schema: <breaking change>.<feature added>.<fix/minor change>
```

Reply:
@Sloox You need to change the version here https://github.com/Flank/flank/blob/b54ce056723a3b95a884ffab16d0d474ac7d45b5/flank-scripts/build.gradle.kts#L31

And then it will work

Reply:
done!